map_model/src/objects/road.rs

```diff
@@ -426,3 +371,73 @@ impl Road {
         )
     }
 }
+
+// TODO All of this is kind of deprecated? During the transiton towards lanes_ltr, some pieces
+// seemed to really need to still handle lanes going outward from the "center" line. Should keep
+// whittling this down, probably. These very much don't handle multiple direction changes.
```

Comment:
`dir_and_offset` is one case. Say we have something like `spd | ds`, meaning `sidewalk, parking, driving going southbound | driving, sidewalk going northbound`. Here, `dir_and_offset` for the parking lane would be `backards, offset 1`. But if we had a two-way cycletrack smushed in there, something like... `spbbd | ds` with orientations ↓↓↑↓↓ | ↑↑, then the parking lane would be... `backwards, offset 3`. The 2 has become a little bit weird, because the parking lane is actually the 4th lane from the center line, but there's a forwards lane smushed in between. `dir_and_offset` doesn't have many callers; one of them is the stuff that comes up with the permanent `EditCmd`. The indexing gets kind of messed up here, and will have to be updated next.

Reply:
Got it, thanks for explaining!